packages/stable/src/__tests__/testUtils.ts

```diff
@@ -186,6 +183,94 @@ const getFileCreateArgs = (
 
   return { postfix, fileContent, sourceCreatedTime, localFileMeta };
 };
+export function setupMockableClientForIntegrationTests() {
+  const client = setupClient(process.env.COGNITE_BASE_URL);
+  return client;
+}
+export function setupLoggedInClientForUnitTest() {
+  return new CogniteClient({
+    appId: 'JS SDK integration tests (beta)',
+    baseUrl: process.env.COGNITE_BASE_URL,
+    project: process.env.COGNITE_PROJECT as string,
+    getToken: () =>
+      login().then((account) => {
+        return account.access_token;
+      }),
+  });
+}
+
+function divideFileIntoChunks(file: string, numChunks: number) {
+  // Read the binary file
+  const fileContentBinary = readFileSync(file);
+
+  // Calculate the number of chunks
+  const chunkSize = Math.ceil(fileContentBinary.length / numChunks);
+
+  // Array to store the chunks
+  const chunks: Buffer[] = [];
+
+  // Divide the file content into chunks
+  for (let i = 0; i < numChunks; i++) {
+    const start = i * chunkSize;
+    const end = start + chunkSize;
+    chunks.push(fileContentBinary.slice(start, end));
+  }
+
+  return chunks;
+}
+function toArrayBuffer(buffer: Buffer) {
```

Comment:
```suggestion

function toArrayBuffer(buffer: Buffer) {
```

Reply:
Just add a empty newline between functions.

Reply:
Or is empty newline between functions not the norm in this file?

Reply:
added spaces. should be with spaces